pyscriptjs/src/main.ts

```diff
@@ -211,8 +190,48 @@ export class PyScriptApp {
            support ES modules in workers:
            https://caniuse.com/mdn-api_worker_worker_ecmascript_modules
         */
-        const interpreterURL = await this.interpreter._remote.src;
+        const interpreterURL = interpreter_cfg.src;
         await import(interpreterURL);
+        return { remote_interpreter, wrapped_remote_interpreter };
+    }
+
+    async _startInterpreter_worker(interpreter_cfg: InterpreterConfig) {
+        logger.warn('execution_thread = "worker" is still VERY experimental, use it at your own risk');
+        logger.info('Starting the interpreter in a web worker');
+        const base_url = this._get_base_url();
+        const worker = new Worker(base_url + '/interpreter_worker.js');
+        const worker_initialize: any = Synclink.wrap(worker);
+        const wrapped_remote_interpreter = await worker_initialize(interpreter_cfg);
+        const remote_interpreter = undefined; // this is _unwrapped_remote
```

Comment:
[question]: we want to kill this eventually right? so the return values of `_startInterpreter_worker` and `_startInterpreter_main` would just change to `wrapped_remote_interpreter` right?

Reply:
yes, we want to kill `_unwrapped_remote` completely at some point